force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -57,9 +76,20 @@ public with sharing class FlowControlLoopsCollections {
     public static Integer findMax(Integer num1, Integer num2) {
         // Initialize largestNum as null
         Integer largestNum = null;
-
-        return null; // Replace null with the variable you used to store the result
-    }
+        if (num1 == null || num2 == null) {
+            return null;
+        }
+            else if  (num1 > num2) {
+                largestNum = num1;
+                return largestNum;
+            }
+            else {
+                largestNum = num2;
+                return largestNum;
```

Comment:
Try using the Math class https://developer.salesforce.com/docs/atlas.en-us.apexref.meta/apexref/apex_methods_system_math.htm#apex_System_Math_max